README.md

```diff
@@ -1 +1,91 @@
-Envoy v2 gRPC APIs
+# Envoy v2 gRPC APIs
+
+## Goals
+
+This repository contains the draft v2 gRPC
```

Comment:
nit: I would make it more clear in first sentence (and even title) that these are both gRPC and JSON APIs, otherwise people will be not pleased until they figure out what we are up to with the dual versions.